code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -200,28 +200,27 @@ can cause issues with ammo types getting mixed up during the burst.
 						/obj/item/attachable/extended_barrel,
 						/obj/item/attachable/compensator,
 						/obj/item/attachable/magnetic_harness,
-						/obj/item/attachable/stock/tactical)
+						/obj/item/attachable/stock/tactical,
+						/obj/item/attachable/verticalgrip,
+						/obj/item/attachable/angledgrip,
+						/obj/item/attachable/attached_gun/flamer,
+						/obj/item/attachable/attached_gun/shotgun,
```

Comment:
The vertical grip makes no sense (not a burst gun), the angled is a very unnecessary buff, the flamer and masterkey were considered but they would need a sprite rework first because they look really bad in it. The gun already comes with an UGL by default.